src/Workload/Microsoft.NET.Sdk.Maui.Manifest/Microsoft.NET.Sdk.Maui.Manifest.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="../Shared/Common.targets" />
 
   <PropertyGroup>
-    <PackageId>$(PackageId).Manifest-$(DotNetMauiManifestVersionBand)</PackageId>
+    <PackageId>$(PackageId)-$(DotNetMauiManifestVersionBand)</PackageId>
```

Comment:
Since `.Manifest` is added to the `.csproj` file name, this seems like the package ID would be the same. 👍